src/views/search/components/search-topbar.module.scss

```
@@ -143,7 +143,7 @@
 		var(--search-page_search-header_padding-vertical)
 		var(--search-page_search-header_padding-horizontal);
 
-	grid-row-gap: var(--search-page_search-header_gap);
+	row-gap: var(--search-page_search-header_gap);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)